eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -109,6 +109,10 @@ class VolumePreparer:
         _, remote_name, ref_name = OSTree.parse_refspec(refspec)
         if not remote_name:
             return None
+
+        # Technically the collection ID configured for updates might not match
+        # the one of the booted OSTree, but we can't use the correct one here
+        # until https://github.com/ostreedev/ostree/issues/1487 is fixed
```

Comment:
I'm not sure I agree this is blocked by that - in the specific case of one commit we're going to read anyway, can't we read the collection-id binding off the commit?

Reply:
yeah, but then `create-usb` would fail since `/ostree/repo/summary` specifies the other (possibly wrong) collection ID along with the ref in question